### ISSUE TITLE
'currentRadioAccessTechnology' is deprecated: first deprecated in macCatalyst 13.0

### ISSUE THREAD

**Issue by charlieporth1** [firebase/firebase-ios-sdk#4956]
I seem to be getting caught while compiling on Xcode 11, for Catalyst.
The Error is as follows
```
'currentRadioAccessTechnology' is deprecated: first deprecated in macCatalyst 13.0
Replace 'currentRadioAccessTechnology' with 'serviceCurrentRadioAccessTechnology'
```
On line 74 of `GDTCORNetworkMobileSubtype GDTCORNetworkMobileSubTypeMessage()` Objective C function in Google Data Transport. 
Objective C is a foreign language to me so I don't know how much help I will be.  It could be a very simple fix, but the syntax is throwing me for a loop.

**Comment by morganchen12** [comment]
It should be a pretty simple fix. Thanks for flagging!

**Comment by mikehaney24** [comment]
Sorry I missed this, this was fixed by #4965
